modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/SelectEntityInput.es.js

```diff
@@ -42,34 +43,26 @@ class SelectEntityInput extends React.Component {
 		} = this.props;
 
 		if (multiple) {
-			AUI().use('liferay-item-selector-dialog', A => {
-				const itemSelectorDialog = new A.LiferayItemSelectorDialog({
-					eventName: id,
-					on: {
-						selectedItemChange: event => {
-							const newVal = event.newVal;
+			const itemSelectorDialog = new ItemSelectorDialog({
+				buttonAddLabel: Liferay.Language.get('select'),
+				eventName: id,
+				title,
+				url: uri
+			});
 
-							if (newVal) {
-								const selectedValues = event.newVal.map(
-									item => ({
-										displayValue: item.name,
-										value: item.id
-									})
-								);
+			itemSelectorDialog.open();
 
-								onChange(selectedValues);
-							}
-						}
-					},
-					strings: {
-						add: Liferay.Language.get('select'),
-						cancel: Liferay.Language.get('cancel')
```

Comment:
Haven't tested the feature just yet, but cancel seems to be gone in the new implementation

Reply:
I see that now cancel comes out of the box